src/components/LuxLibraryFooter.vue

```diff
@@ -49,7 +49,7 @@ export default {
   },
   methods: {
     value: function (theme) {
-      return theme == "light" ? "light" : "dark"
+      return theme == "light" ? "light" : theme == "shade" ? "shade" : "dark"
```

Comment:
This line is getting a little complex for me.  Could we do something like this instead?
```suggestion
      if (theme == "light" || theme == "shade") { return theme }
      return "dark"
```